_docs/index.md

```diff
@@ -10,9 +10,11 @@ type: markdown
 ---
 {% include home.html %}
 
-# Welcome
```

Comment:
I changed it to match what you asked in #406 and to make it extra clear this set is potentially put of date.